rethinkdb/asyncio_net/net_asyncio.py

```diff
@@ -202,7 +202,7 @@ def connect(self, timeout):
             self._streamreader, self._streamwriter = yield from asyncio.open_connection(
                 self._parent.host,
                 self._parent.port,
-                loop=self._io_loop,
+                #loop=self._io_loop, droped at py3.10
```

Comment:
This makes the driver backward incompatible with older python versions.

Reply:
It does?  How so?  The parameter was optional and otherwise it uses the active loop.